internal/resource/mappers/bimapper/bimapper.go

```diff
@@ -157,16 +191,58 @@ func (m *Mapper) LinksForItem(item *pbresource.ID) []*pbresource.Reference {
 	return out
 }
 
+// LinkIDsForItem returns IDs to links related to the requested item.
+func (m *Mapper) LinkIDsForItem(item *pbresource.ID) []*pbresource.ID {
+	if !resource.EqualType(item.Type, m.itemType) {
+		panic(fmt.Sprintf("expected item type %q got %q",
+			resource.TypeToString(m.itemType),
+			resource.TypeToString(item.Type),
+		))
+	}
+
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
+	links, ok := m.itemToLink[resource.NewReferenceKey(item)]
+	if !ok {
+		return nil
+	}
+
+	out := make([]*pbresource.ID, 0, len(links))
+	for l := range links {
+		out = append(out, l.ToID())
+	}
+	return out
+}
+
 // ItemsForLink returns item ids for items related to the provided link.
+// Deprecated: use ItemIDsForLink
```

Comment:
Is there a reason to keep this around too?

Reply:
It makes the other PR chains land without too much issue is the main one.

Reply:
Ahh, makes sense. Thank you